integration-cli/docker_cli_swarm_test.go

```diff
@@ -1079,10 +1079,9 @@ func (s *DockerSwarmSuite) TestSwarmLockUnlockCluster(c *check.C) {
 	c.Assert(getNodeStatus(c, d2), checker.Equals, swarm.LocalNodeStateActive)
 
 	// once it's caught up, d2 is set to not be locked
-	c.Assert(d2.Restart(), checker.IsNil)
```

Comment:
should fix https://jenkins.dockerproject.org/job/Docker%20Master/label=fedora23%20&&%20devicemapper/7771/console